tests/unit/pypyr/pipelinerunner_test.py

```diff
@@ -18,7 +18,6 @@ def mock_parser(context_arg):
 
 def mock_parser_none(context_arg):
     """Return None, mocking get_parsed_context"""
-    print('FUCK YOU CUNT')
```

Comment:
Personally I blame the reviewer, 100% 

I had a chat with the coder and he says he's fully aware it's open source, it's just it was late and a mock wasn't doing what he thought it should be doing  and he wanted to print an obvious placeholder to the output for troubleshooting. And then forgot to take it out again.